bazel-mode.el

```diff
@@ -33,6 +33,13 @@
   :link '(url-link
           "https://github.com/bazelbuild/buildtools/tree/master/buildifier"))
 
+(defcustom bazel-mode-buildifier-before-save nil
+  "Specifies whether to run buildifer in 'before-save-hook'."
+  :type 'boolean
+  :group 'bazel-mode
+  :link '(url-link
```

Comment:
This should probably be marked as risky since buildifier is an external tool that could do anything.

Reply:
Done. I also changed bazel-mode-buildifier-cmd to bazel-mode-buildifier-command as https://www.gnu.org/software/emacs/manual/html_node/elisp/File-Local-Variables.html indicates this will automatically mark it as risky as well.